examples/tagmap/tagmap-layer/tagmap-layer.js

```diff
@@ -0,0 +1,113 @@
+/* global document, window */
+/* eslint-disable max-len */
+import {CompositeLayer, IconLayer, WebMercatorViewport} from 'deck.gl';
+import {makeTextureAtlasFromLabels} from './label-utils';
+import {tagmapLayout} from './tagmap-utils';
+import colorbrewer from 'colorbrewer';
+
+/* Constants */
+const defaultProps = {
+  getLabel: x => x.label,
+  getWeight: x => x.weight,
+  getPosition: x => x.coordinates,
+  // ['#1d91c0', '#41b6c4', '#7fcdbb', '#c7e9b4', '#edf8b1']
+  colorScheme: colorbrewer.YlGnBu[9].slice(1, 6).reverse(),
+  minFontSize: 14,
+  maxFontSize: 32,
+  weightThreshold: 1
+};
+
+export default class TagmapLayer extends CompositeLayer {
+
+  initializeState() {
+    // create canvas for measuring the text size
+    const canvas = document.createElement('canvas');
```

Comment:
so we are creating a canvas here and also in makeTextureAtlasFromLabels() function. I know one of them is used for rendering text labels into textures, what is this canvas used fore?